contracts/GitHubUsernames.sol

```diff
@@ -4,7 +4,8 @@ pragma solidity ^0.8.19;
 import {IPassportUtils} from "./utils/IPassportUtils.sol";
 
 contract GitHubUsernames {
```

Comment:
asking to learn, why not extend Ownable..? the setOwner seems to be duplicating in all contracts..

Reply:
@kishore881 Just to keep the contracts lightweight and to keep the gas costs at a minimum.  But we haven't done a comparison of the gas costs with/without Ownable yet.